src/StreetNameRegistry/Municipality/Exceptions/StreetNameCorrectionExceededChangeLimitException.cs

```diff
@@ -0,0 +1,27 @@
+namespace StreetNameRegistry.Municipality.Exceptions
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    public sealed class StreetNameCorrectionExceededChangeLimitException : StreetNameRegistryException
```

Comment:
Idk about the name, limit suggests we can offer a limit in the exception which would be useful. The distance `30` doesn't seem useful for the client, which is ok, this shouldn't be for the client to know.
Also it's the streetname name.

`StreetNameNameCorrectionLevenshteinException` ? At least it's clear why this exception occurred.
Or something else.

Reply:
I actually included Levensthein in the first version of the exception but then went for something more generic because we could choose to change the algo and it shouldn't have an impact on the general domain concept of having a change limit.

Reply:
I added Name to the exception but I'm not sure what you want it to look like if not using ChangeLimit like I have now, it makes sense to me given that it describes exactly why the exception is occurring.

Reply:
ChangeLimit doesn't say anything for me. Change name sure, but which limit?
I think you're looking more for `StreetNameNameCorrectionExceededCharacterLimit` or something like that. Then we know it's about the characters in the name.

Reply:
The name is not perfect yet because it could imply that there are too many characters.

Reply:
ChangeLimit implies that there is a certain limit to how much a name can be changed.

Reply:
But what limit? The names in the street? The Names within the streetname? We both know it's characters because we looked at the code.

Reply:
CharacterChangeLimit then?